packages/blocks/src/database-block/table/components/selection/cell-selection.ts

```diff
@@ -39,6 +39,12 @@ export class CellLevelSelection extends WithDisposable(LitElement) {
     this.state = null;
   };
 
+  private get _zoom() {
```

Comment:
Before this PR, no `zoom` or `edgeless` keyword exists in the database block. Is there some way that the block UI can simply fit in without special adaptation to the whiteboard state?